packages/core/src/domain/configuration/configuration.ts

```diff
@@ -144,6 +144,11 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
     return
   }
 
+  if (initConfiguration.version && typeof initConfiguration.version !== 'string') {
+    display.error('Version must be defined as a string')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IIUC, the error happens when building tags for the intake requests, so should we do the same kind of validation for other customer provided tags? (i.e. service and env)

Reply:
👍 After testing, the same issue occurs for service and env tag parameter. I added them to the check.